Source: http://git.gnome.org/cgit/eog/commit/?id=410444769709a68fd2049786c46926f1e1b30e53
Upstream: Yes
Reason: Eog fails to compile if exempi/exif are not available
--- a/src/eog-properties-dialog.c
+++ b/src/eog-properties-dialog.c
@@ -530,6 +530,7 @@ eog_properties_dialog_set_netbook_mode (EogPropertiesDialog *dlg,
 
 	priv->netbook_mode = enable;
 
+#ifdef HAVE_METADATA
 	if (enable) {
 		gtk_widget_reparent (priv->metadata_details_sw,
 				     priv->metadata_details_box);
@@ -547,6 +548,7 @@ eog_properties_dialog_set_netbook_mode (EogPropertiesDialog *dlg,
 			gtk_notebook_prev_page (GTK_NOTEBOOK (priv->notebook));
 		gtk_widget_hide_all (priv->metadata_details_box);
 	}
+#endif
 }
 
 static void
